app/controllers/spree/admin/base_controller.rb

```diff
@@ -21,18 +22,6 @@ def warn_invalid_order_cycles
         flash[:notice] = warning if warning.present?
       end
 
-      # This is in Spree::Core::ControllerHelpers::Auth
-      # But you can't easily reopen modules in Ruby
-      def unauthorized
-        if spree_current_user
-          flash[:error] = t(:authorization_failure)
```

Comment:
There's a failing spec related to this line. The error uses `t(:authorization_failure)` here but in the module version it uses `Spree.t(:authorization_failure)` which is sightly different. We could maybe update the module version to use the non-spree translation and delete the I18n key?

Reply:
done!

I'm thinking that removing the whole Spree.t and its keys wouldn't be too difficult but that would require syncing all the matching locales as well.